### PR TITLE
ESRP codesign master branch

### PR DESCRIPTION
**Pull request by vladimir-shcherbakov**

### CODE REVIEW

File: build.proj [hunk 1]
```diff
@@ -51,7 +51,7 @@
     <LocalFeedFolder>$(LibraryToolsFolder)\LocalFeed</LocalFeedFolder>
     <PublishDirectory>$(LibrarySourceFolder)\Publish</PublishDirectory>
     <Configuration Condition=" '$(Configuration)' != 'Release'">Debug</Configuration>
-    <CodeSign Condition=" '$(CodeSign)' == '' ">false</CodeSign>
+    <CodeSign>false</CodeSign>
```
Comment (markcowl) [inline build.proj]: Same comment as in the PR below, this needs to be conditional so we can sign bits

Comment (vladimir-shcherbakov) [reply build.proj]: I don't get the comment. To change the default ```CodeSign``` property  value we can use ```/p:CodeSign=true``` switch.

## Experiment 
Targets file  ```DoSomething.targets``` content:
```
<?xml version="1.0" encoding="utf-8"?>
<Project DefaultTargets="Action" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">

    <PropertyGroup>
        <CodeSigh>false</CodeSigh>
    </PropertyGroup>

    <Target Name="Action">
        <!-- This is a built-in task -->
        <Message Text="CodeSigh value is: $(CodeSigh)" />
    </Target>
</Project>
```
Show property ```CodeSigh``` default  value:
```
E:\temp>msbuild DoSomething.targets /t:Action
Microsoft (R) Build Engine version 15.8.169+g1ccb72aefa for .NET Framework
Copyright (C) Microsoft Corporation. All rights reserved.

Build started 12/18/2018 8:30:37 PM.
Project "E:\temp\DoSomething.targets" on node 1 (Action target(s)).
Action:
  CodeSigh value is: false
Done Building Project "E:\temp\DoSomething.targets" (Action target(s)).


Build succeeded.
    0 Warning(s)
    0 Error(s)

Time Elapsed 00:00:00.05
```
Change property ```CodeSigh``` value to true:
```
E:\temp>msbuild DoSomething.targets /t:Action /p:CodeSigh=true
Microsoft (R) Build Engine version 15.8.169+g1ccb72aefa for .NET Framework
Copyright (C) Microsoft Corporation. All rights reserved.

Build started 12/18/2018 8:33:20 PM.
Project "E:\temp\DoSomething.targets" on node 1 (Action target(s)).
Action:
  CodeSigh value is: true
Done Building Project "E:\temp\DoSomething.targets" (Action target(s)).


Build succeeded.
    0 Warning(s)
    0 Error(s)

Time Elapsed 00:00:00.05
```

**Comment by markcowl** [comment]
signing: https://azuresdkci.westus2.cloudapp.azure.com/view/PowerShell/job/psnetsign-esrp/8/

**Comment by markcowl** [comment]
signing run here: https://azuresdkci.westus2.cloudapp.azure.com/view/PowerShell/job/psnetsign-esrp/9/

**Comment by markcowl** [comment]
Closing in favor of #8193

**Pull request closed without merging**